configs/default/FOXE-FOXEERF722V4.config

```diff
@@ -8,6 +8,7 @@
 #define USE_ACC_SPI_ICM42688P
 #define USE_BARO
 #define USE_BARO_DPS310
+#define USE_BARO_BMP280
```

Comment:
This change is already merged into master (https://github.com/betaflight/unified-targets/commit/73961c69f9a366725c063b6af61a795772484f40)

```suggestion
```